deployment_scripts/deploy_windows.ps1

```diff
@@ -9,6 +9,21 @@ param(
     [Bool]
     $agents = $true
 )
+
+function Configure-precommit([String] $python_command)
```

Comment:
My only question, and this is due to my ignorance about powershell, is whether or not the `pre-commit install` command is being run in the appropriate directory.

Reply:
I guess this indicates that the current directory is the top of the git repo?
`$monkey_home = (Get-Item -Path ".\").FullName`